ccloud/docs/includes/set-acls-origin-and-destination.rst

```diff
@@ -0,0 +1,35 @@
+|crep| must have authorization to read Kafka data from the origin cluster and write Kafka data in the destination |ccloud| cluster.
+|crep| should be run with a |ccloud| service account, not super user credentials, so use |ccloud| CLI to configure appropriate ACLs for the service account id corresponding to |crep|.
+Since the origin cluster and destination cluster in this example are both |ccloud|, configure appropriate ACLs for the service account ids corresponding to |crep| in both |ccloud| clusters.
+For more details on |crep| ACLs, see :ref:`replicator_security_overview`.
+
+In the origin |ccloud|:
+
+.. sourcecode:: bash
+
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation CREATE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation WRITE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation READ --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE-CONFIGS --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation ALTER-CONFIGS --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE --cluster-scope
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation CREATE --cluster-scope
```

Comment:
Unsure if there's a different use case I'm missing, but for Replicator, if there's already `CREATE` ACLs on the source topic, is `CREATE` on the cluster still necessary? Also we might not need the exact same permissions on the source as we do on the destination (like `ALTER-CONFIGS`), depending on what exactly the workflow here is.

Reply:
@skaundinya15 

1. wrt `ALTER-CONFIGS`, good catch:I have removed `ALTER-CONFIGS` from the source
2. wrt "is CREATE on the cluster still necessary" --  IIUC it is documented as required on this page: https://docs.confluent.io/current/multi-dc-deployments/replicator/index.html#security "If using the topic creation and config sync features of Replicator (enabled by default):"    WDYT?

Reply:
With respect to number 2, I checked that and realized that was incorrect. I merged in #4371 to correct this issue so now it should reflect the correct ACLs needed to run Replicator.

Reply:
👍 thanks for https://github.com/confluentinc/docs/pull/4731/files